specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/job.json

```diff
@@ -0,0 +1,931 @@
+                }
+            }
+        },
+        "JobID": {
+            "description": "The object with the Job ID.",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "format": "int64",
+                    "x-nullable": false,
+                    "description": "The job number."
+                },
+                "jtIdentifier": {
+                    "type": "string",
+                    "description": "The jobTracker identifier."
+                }
+            }
+        },
+        "JobListJsonObject": {
+            "description": "The List Job operation response.",
+            "properties": {
+                "detail": {
+                    "$ref": "#/definitions/JobDetailRootJsonObject",
+                    "description": "The detail of the job."
+                },
+                "id": {
+                    "type": "string",
+                    "description": "The Id of the job."
+                }
+            }
+        },
+        "JobSubmissionJsonResponse": {
+            "description": "The job submission json response.",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "The Id of the created job."
+                }
+            }
+        },
+        "Profile": {
+            "description": "The object containing the job profile information.",
+            "properties": {
+                "jobFile": {
+                    "type": "string",
+                    "description": "The job configuration file."
+                },
+                "jobId": {
+                    "type": "string",
+                    "description": "The full ID of the job."
+                },
+                "jobID": {
+                    "$ref": "#/definitions/JobID",
+                    "description": "The ID of the job."
+                },
+                "jobName": {
+                    "type": "string",
+                    "description": "The user-specified job name."
+                },
+                "queueName": {
+                    "type": "string",
+                    "description": "The name of the queue to which the job is submitted."
+                },
+                "url": {
+                    "type": "string",
+                    "description": "The link to the web-ui for details of the job."
+                },
+                "user": {
+                    "type": "string",
+                    "description": "The userid of the person who submitted the job."
+                }
+            }
+        },
+        "Status": {
+            "description": "Gets or sets the object containing the job status information.",
+            "properties": {
+                "cleanupProgress": {
+                    "type": "number",
+                    "format": "double",
+                    "x-nullable": false,

```

Comment:
There is no need to specify x-nullable false. This is the default.

Reply:
This is not default for basic types in the generated C# code . As we disscussed offline, I will remove all `"x-nullable": false`  in the next iteration.

---
In reply to: [237732181](https://github.com/Azure/azure-rest-api-specs/pull/4489#discussion_r237732181) [](ancestors = 237732181)